src/main/java/org/jabref/gui/collab/DatabaseChangesResolverDialog.java

```diff
@@ -36,6 +38,7 @@ public class DatabaseChangesResolverDialog extends BaseDialog<Boolean> {
      */
     private final Map<DatabaseChange, DatabaseChangeDetailsView> DETAILS_VIEW_CACHE = new HashMap<>();
 
+
```

Comment:
Strange that checkstyle did not complain because of two empty lines 😋

Reply:
because not in class?